tests/constraints.txt

```diff
@@ -1,9 +1,9 @@
 importlib-metadata==0.22
 packaging==19.0
-pep517==0.9.1
+pyproject_hooks==1.0
 setuptools==42.0.0; python_version < "3.10"
 setuptools==56.0.0; python_version >= "3.10"
 toml==0.10.0
-tomli==1.0.0
+tomli==1.1.0
```

Comment:
This should be equal to the minimum version specified in `pyproject.toml`. Did you experience an issue with 1.0.0?